drivers/hmis/lib/form_data/static/client_alert.json

```diff
@@ -0,0 +1,38 @@
+{
+  "item": [
+    {
+      "type": "CHOICE",
+      "link_id": "priority",
+      "text": "Alert Priority Level",
+      "pick_list_reference": "ClientAlertPriorityLevel",
+      "mapping": {
+        "field_name": "priority"
+      }
+    },
+    {
+      "type": "TEXT",
+      "link_id": "note",
+      "text": "Alert Note",
+      "mapping": {
+        "field_name": "note"
+      }
+    },
+    {
+      "type": "DATE",
```

Comment:
Just confirming, the more elaborate date/duration picker component in the figma file is a non mvp requirement, correct?

And just out of curiosity, once we do take that on, how will we incorporate it since the form is generated from these json files? Will we need to create a new form type?

Reply:
Yes, non mvp. Since it ultimately collects a date value, I think it would be a new "component" we would need to add. Defined something like `"type": "DATE", "component": "DURATION_PICKER"`. The "type" should more or less define the data type thats collected, while the "component" is used to specify which presentational element to use (if different from the default). I think it would fall into this category but it might need some more thought.